### PR TITLE
Edge based services in proxy

### PR DESCRIPTION
**Pull request by wojtek-t**
This is sibling effort to what I did for endpoints in KubeProxy.
This PR is first one (changing config & iptables) - userspace will follow.

### CODE REVIEW

File: pkg/proxy/config/config.go [hunk 1]
```diff
@@ -45,7 +45,24 @@ type ServiceConfigHandler interface {
 	OnServiceUpdate(services []*api.Service)
 }
 
-// EndpointsHandler is an abstract interface o objects which receive
+// ServiceHandler is an abstract interface of objects whic receive
```
Comment (ncdc) [inline pkg/proxy/config/config.go]: typo: which

Comment (wojtek-t) [inline pkg/proxy/config/config.go]: done

**Comment by wojtek-t** [comment]
@thockin - PTAL (I'm going to send a similar PR to https://github.com/kubernetes/kubernetes/pull/44494 for services as a follow up to this).

**Comment by wojtek-t** [comment]
@thockin - any reason why you closed this?

**Comment by wojtek-t** [comment]
@k8s-bot etcd3 test this, issue: #43520

**Pull request merged**